.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Source code
+*.kt            text eol=lf
+*.java          text eol=lf
+*.xml           text eol=lf
+
+# Gradle files
+*.gradle        text eol=lf
+*.properties    text eol=lf
+*.bat           text eol=crlf
```

Comment:
Something is fishy still, even after resetting gradlew.bat keeps changing, any ideas welcome.

I tried autocrlf=true and false, and then reset, but no luck.

Reply:
Re-cloned and all good, but I was able to reproduce it with the code here: https://stackoverflow.com/q/48642692/253468

This however fixes it: https://stackoverflow.com/a/19588910/253468

Right now it's above my git abilities, but it should be fine considering retrofit uses the same file.

Reply:
It was solved in: https://github.com/cashapp/paparazzi/pull/721